daemon/c_cgroups.c

```diff
@@ -996,14 +1016,26 @@ c_cgroups_devices_chardev_allow(void *cgroupsp, int major, int minor, bool assig
 }
 
 static int
-c_cgroups_devices_chardev_deny(void *cgroupsp, int major, int minor)
+c_cgroups_devices_dev_deny(void *cgroupsp, char type, int major, int minor)
 {
 	c_cgroups_t *cgroups = cgroupsp;
 	ASSERT(cgroups);
 
 	int ret;
+	char dev_type;
```

Comment:
this needs to be int and the switch case should map char to int (case b -> DEVCG_TYPE_BLOCK
int type_int analogous  to c_cgroups_devices_is_dev_allowed()

Reply:
done